### DISCUSSION TITLE
Parallel Installation

### DISCUSSION THREAD

**Discussion by TimBoWi** [camueller/SmartApplianceEnabler#263] [Installation und Betrieb]
Hallo zusammen, 

Erstmal ein starkes Projekt und vielen Dank für deine Mühen Axel. Ich werde auch ein Beitrag für deine Mühen spenden. Danke für deinen Pioniergeist und deinen fachlichen Input. 

Ich werde zum Ende des Jahres einen DIY Akku bauen und möchte daher jetzt schon mal ein paar Geräte über meinen SHM integrieren. 

Mir stellt sich gerade die Frage, ob ich auf meinem Pi4 SAE einfach parallel zu HomeBridge installieren kann über die manuelle Installation. 
Aktuell es ist es sehr schwierig an einen neuen PI zu kommen, daher dachte ich daran SAE parallel aufzuspielen. 

VG
Tim

**Comment by camueller** [comment]
Danke für Deinen Beitrag via Paypal :-)

Eigentlich spricht nichts gegen eine Parallelinstallation - die Ports sind konfigurierbar.
